frontend/src/app/app.component.html

```diff
@@ -0,0 +1,5 @@
+<div>
+	<ul>
+		<app-list *ngFor="let todo of todos_array[0]" [single_todo]="todo"></app-list>
```

Comment:
this will cause JavaScript error if empty has no value... potential nullpointer exception

Reply:
Sorry I don't understand, can you be more specific